### ISSUE TITLE
classDiagram Not working?

### ISSUE THREAD

**Issue by seigel** [inkdropapp/inkdrop-mermaid#1]
I seem to get this error with the latest plugin version:

```Cannot read property 'members' of undefined```

This is the code I am using:

```mermaid 
classDiagram
  class BankAccount
  BankAccount : +String owner
  BankAccount : +Bigdecimal balance
  BankAccount : +deposit(amount)
  BankAccount : +withdrawl(amount)    
```

**Comment by craftzdog** [comment]
Thank you for the report.
Seems like mermaid version is old in the plugin.
I'll work on it.

**Comment by craftzdog** [comment]
Fixed in v2.1.0.
https://my.inkdrop.app/plugins/mermaid
Thanks!

**Comment by seigel** [comment]
Thanks sir!
